packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -680,6 +697,18 @@ export class GarbageCollector implements IGarbageCollector {
         return builder.getSummaryTree();
     }
 
+    public getMetadata(): IGCMetadata {
```

Comment:
@sonalivdeshpande Looks like the code that added this to the `IGarbageCollector` interface was missed during merge. Take a look at https://github.com/microsoft/FluidFramework/pull/10173/files to see where it should be added.